howso/get_cases.amlg

```diff
@@ -193,71 +208,59 @@
 		))
 
 		;output a map with features and cases
-		(assoc
-			features features
-			cases
-				(if (and hasEncodedFeatures (not skip_decoding))
-					;decode nominal features if necessary
-					(map
-						(lambda
-							(call ConvertToOutput (assoc
-								features features
-								feature_values (current_value 1)
-								has_rounded_features has_rounded_features
-								has_datetime_features has_datetime_features
-							))
-						)
-						case_values_list
-					)
+		(call !Return (assoc
+			payload
+				(assoc
+					features features
+					cases
+						(if (and !hasEncodedFeatures (not skip_decoding))
+							;decode nominal features if necessary
+							(map
+								(lambda
+									(call !ConvertToOutput (assoc
+										features features
+										feature_values (current_value 1)
+										has_rounded_features has_rounded_features
+										has_datetime_features has_datetime_features
+									))
+								)
+								case_values_list
+							)
 
-					;else round if needed
-					has_rounded_features
-					(map
-						(lambda (call RoundContinuousFeatures (assoc features features feature_values (current_value 1) )))
-						case_values_list
-					)
+							;else round if needed
```

Comment:
GH shows weird diff on the spaces leading up to this comment, maybe its a spaces/tabs thing?

Reply:
goes down some lines too - to the end of the map under `has_rounded_features`

Reply:
this is a GH issue. i'll fix whatever other whitespace issues are in the file while i'm at it though